mitmproxy/contentviews/__init__.py

```diff
@@ -173,11 +43,23 @@ def get_content_view(viewmode: View, data: bytes, **metadata):
 add(image.ViewImage())
 add(query.ViewQuery())
 
-if protobuf.ViewProtobuf.is_available():
+if protobuf.ViewProtobuf.is_available():  # pragma: no cover
```

Comment:
Regardless of the protobuf-view quality, it was never available - I guess travis misses a package or something? Or does this only work if one manually installs the package?